src/components/localization/config.js

```diff
@@ -27,4 +27,20 @@ i18n.languages = Object.keys(language_config)
 
 export const localize = (string, values) => t(crc32(string), { defaultValue: string, ...values })
 
+/**
+ * @description if you need the direction of the current languge please use this fucntion ( without react life-cycles), if you need react life-cycles please use useLangDirection hook
+ * @returns {"ltr" | "rtl"} the current language direction
+ */
+export const get_lang_direction = () => {
+    return i18n.dir(i18n.language)
```

Comment:
do we need to pass the language here since we want to get the dir of the current language?

Reply:
Thanks @farrah-deriv , I can remove this. cool :pray: